index.html

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;500;700&display=swap">
+    <title>Portfolio page </title>
+</head>
+<body>
+    <div class="main-one">
+        <header>
+            <h1 class="mylogo">Ndonyi</h1>
+            <menu>
+                <ul class="home">
+                    <li class="content"><a href="#">Portoflio</a></li>
+                    <li class="content"><a href="https://citec-47.github.io/frontendmasters/">About</a></li>
```

Comment:
You should also avoid lines of code that are too long (100 characters or more). [check here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=Avoid%20lines%20of%20code%20that%20are%20too%20long%20(100%20characters%20or%20more).)